install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -423,10 +421,10 @@ data:
     {{.}}
   {{- end }}
 {{- end }}
-{{ if and .Values.global.hubble.ui.enabled (not (has "0.0.0.0:50051" .Values.global.hubble.listenAddresses)) }}
-  # A space separated list of additional addresses for Hubble server to listen to.
+  # A space separated list of additional addresses for Hubble server to listen to (e.g. ":50051 :50052").
+{{- if and .Values.global.hubble.ui.enabled (not (has "0.0.0.0:50051" .Values.global.hubble.listenAddresses)) }}
```

Comment:
well actually it's more natural for the hubble-ui chart to check if `.Values.global.hubble.listenAddresses` is set and pick the first port from there or something. maybe in a separate PR though.

Reply:
or rather the ui should point to the relay service instead. i'll create a separate PR for that.

Reply:
Yes, Hubble UI should target hubble-relay.
I don't actually see a reason to expose the hubble server on multiple addresses. We probably want to simplify this before v1.8.

Reply:
https://github.com/cilium/cilium/pull/11330